maestro/Makefile

```diff
@@ -0,0 +1,7 @@
+# Make file for local development
```

Comment:
we usually don't include Makefile inside chart, reference official helm charts https://github.com/helm/charts/tree/master/stable/

Reply:
same as above